src/main/java/com/contentful/java/cma/ModuleContentTypes.java

```diff
@@ -167,15 +164,24 @@ public CMAContentType unPublish(CMAContentType contentType) {
   public CMAContentType update(CMAContentType contentType) {
     assertNotNull(contentType, "contentType");
     assertNotNull(contentType.getName(), "contentType.name");
+
     String contentTypeId = getResourceIdOrThrow(contentType, "contentType");
     String spaceId = getSpaceIdOrThrow(contentType, "contentType");
     Integer version = getVersionOrThrow(contentType, "update");
-    return service.update(
-        version,
-        spaceId,
-        contentTypeId,
-        contentType
-    ).toBlocking().first();
+
+    final CMASystem system = contentType.getSystem();
+    contentType.setSystem(null);
+
+    try {
+      return service.update(
+          version,
+          spaceId,
+          contentTypeId,
+          contentType
+      ).toBlocking().first();
+    } finally {
```

Comment:
@realityking the `sys` field will always be set back. The update will be returning a new object and I need to make sure the handed in object is not changed on return. If I wouldn't change the system to null, it would get serialised for no reason.

@loudmouth I am not sure If I understand. the finally block will get executed either if there is an exception or not.